include/umf/memory_pool_ops.h

```diff
@@ -36,9 +36,8 @@ typedef struct umf_memory_pool_ops_t {
 
     ///
     /// @brief Initializes memory pool.
-    /// @param providers array of memory providers that will be used for coarse-grain allocations.
+    /// @param provider memory provider that will be used for coarse-grain allocations.
     ///        Should contain at least one memory provider.
```

Comment:
so this `Should contain at least one memory provider.` sentence could be removed as well, I guess

Reply:
Done, thanks